source/features/add-releases-tab.js

```diff
@@ -1,46 +1,52 @@
-import select from 'select-dom';
+/*
+Access a repository’s releases using the Releases tab or by pressing g r.
+https://cloud.githubusercontent.com/assets/170270/13136797/16d3f0ea-d64f-11e5-8a45-d771c903038f.png
+
+The tab isn’t shown if there are no releases.
+*/
+
 import {h} from 'dom-chef';
+import select from 'select-dom';
 import * as icons from '../libs/icons';
 import * as pageDetect from '../libs/page-detect';
 import {registerShortcut} from './improve-shortcut-help';
 
 const repoUrl = pageDetect.getRepoURL();
-const repoKey = `${repoUrl}-releases-count`;
+const repoKey = `releases-count-${repoUrl}`;
```

Comment:
This was changed because it was stored as a `string`, now it's a `number`. It just makes it easier to deal with it.